Specs/NumberWithUnit/English/CurrencyModel.json

```diff
@@ -1924,5 +1924,300 @@
+      {
+        "Text": "10 $",
+        "Start": 0,
+        "End": 3,
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Dollar",
+          "value": "10"
+        }
+      },
+      {
+        "Text": "10 €",
+        "Start": 25,
+        "End": 28,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "10"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Banknotes: 5 €, 10 €, 20 €, 50 €, 100 €, 200 €, 500 €.",
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "5 €",
+        "Start": 11,
+        "End": 13,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "5"
+        }
+      },
+      {
+        "Text": "10 €",
+        "Start": 16,
+        "End": 19,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "10"
+        }
+      },
+      {
+        "Text": "20 €",
+        "Start": 22,
+        "End": 25,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "20"
+        }
+      },
+      {
+        "Text": "50 €",
+        "Start": 28,
+        "End": 31,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "50"
+        }
+      },
+      {
+        "Text": "100 €",
+        "Start": 34,
+        "End": 38,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "100"
+        }
+      },
+      {
+        "Text": "200 €",
+        "Start": 41,
+        "End": 45,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "200"
+        }
+      },
+      {
+        "Text": "500 €",
+        "Start": 48,
+        "End": 52,
+        "TypeName": "currency",
+        "Resolution": {
+          "isoCurrency": "EUR",
+          "unit": "Euro",
+          "value": "500"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "1 $10 dollars bill",
+    "Comment": "This is to show the current behaviour.",
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "1 $",
+        "Start": 0,
+        "End": 2,
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Dollar",
+          "value": "1"
+        }
+      },
+      {
+        "Text": "10 dollars",
+        "Start": 3,
+        "End": 12,
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Dollar",
+          "value": "10"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "1 $10 dollars bill",
+    "Comment": "This is to show the ideally behaviour.",
```

Comment:
Why would this one fail? My understanding was that "1 $ 10 dollars bill" below is the ambiguous case.